andela/activo-web
# #158560264 User can save new asset

**What does this PR do?**
Adds the functionality that enables a user to add an asset

**Description of Task to be completed?**
- create method handler for handling requests from client
- create actions and action creators responsible for updating the store on successful asset addition
- write tests to test implementation of method handlers

**How should this be manually tested?**
After cloning the repo or fetching the branch, run the application.
- Click on the add asset button on the dashboard to open the add asset pane
- Select the Asset category of your choice
- Fill in the fields and click on `Add Asset`
- If successful, you should get a success toast confirming your successful creation of an asset.

**Any background context you want to provide?**
N/A

**What are the relevant pivotal tracker stories?**

[#158560264](https://www.pivotaltracker.com/story/show/158560264)

**Screenshots (if appropriate)**
<img width="1440" alt="screen shot 2018-08-01 at 9 42 01 am" src="https://user-images.githubusercontent.com/24798364/43520703-859d5342-958b-11e8-9ee5-fd9c42de437a.png">

--------------------------------------------------------------

<img width="1440" alt="screen shot 2018-08-01 at 9 42 05 am" src="https://user-images.githubusercontent.com/24798364/43520718-8f836d10-958b-11e8-85be-3df3c5848f51.png">



**Questions:**

N/A
